docs/index.md

```diff
@@ -1,8 +1,8 @@
-# playing-the-internship-game
+# Playing the Internship Game
 
-Content created by [Evy Kassirer](http://www.evykassirer.com/)
+Content created by [Evy Kassirer](http://www.evykassirer.com/) and [Rudi Chen](http://digitalfreepen.com/)
 
-Also viewable at [this website](https://evykassirer.github.io/playing-the-internship-game)
+You can view the source markdown files in the Docs folder. This content is also viewable at [this website](https://evykassirer.github.io/playing-the-internship-game)
 
 ### The rules, tips, and tricks I've learned on how to approach tech internships
```

Comment:
You mean like this?

`### A note from Evy: The rules, tips, and tricks I've learned on how to approach tech internships`